tox.ini

```diff
@@ -1,31 +1,29 @@
 [tox]
 envlist=
     docs,
-    py27-django{111},
-    py35-django{111,20,21},
-    py36-django{111,20,21},
+    py310-django{32}
 
 [testenv]
 changedir=
     oidc_provider
 deps =
     mock
-    psycopg2
-    pytest==3.6.4
-    pytest-django
-    pytest-flake8
-    pytest-cov
-    django111: django>=1.11,<1.12
-    django20: django>=2.0,<2.1
-    django21: django>=2.1,<2.2
+    psycopg2-binary
+    pytest==7.2.2
+    pytest-flake8==1.0.7,
+    pytest-cov==4.0.0
+    pytest-dotenv==0.5.2
+    pytest-django==4.5.2
+    pytest-xdist==3.2.1
+    django32: django==3.2.18
 
 commands =
-    pytest --flake8 --cov=oidc_provider {posargs}
+    pytest --cov=oidc_provider {posargs}
 
 [testenv:docs]
-basepython = python2.7
+basepython = python3.10
 changedir = docs
-whitelist_externals =
+allowlist_externals =
     mkdir
 deps =
     sphinx
```

Comment:
tried to match what chassis will use so that were working with a common base.
drop python 2 / django 1 & 2 support.